boards/arm/nucleo_l073rz/doc/nucleol073rz.rst

```diff
@@ -0,0 +1,168 @@
+.. _nucleo_l073rz_board:
+
+ST Nucleo L073RZ
+################
+
+Overview
+********
+The STM32 Nucleo-64 development board with STM32L073RZ MCU, supports Arduino and ST morpho connectivity.
+
+The STM32 Nucleo board provides an affordable, and flexible way for users to try out new concepts,
+and build prototypes with the STM32 microcontroller, choosing from the various
+combinations of performance, power consumption, and features.
+
+The Arduino* Uno V3 connectivity support and the ST morpho headers allow easy functionality
+expansion of the STM32 Nucleo open development platform with a wide choice of
+specialized shields.
+
+The STM32 Nucleo board integrates the ST-LINK/V2-1 debugger and programmer.
+
+The STM32 Nucleo board comes with the STM32 comprehensive software HAL library together
+with various packaged software examples.
+
+.. image:: img/nucleo_l073rz_board.jpg
+     :width: 500px
+     :height: 367px
+     :align: center
+     :alt: Nucleo L073RZ
+
+More information about the board can be found at the `Nucleo L073RZ website`_.
+
+Hardware
+********
+Nucleo L073RZ provides the following hardware components:
+
+- STM32 microcontroller in QFP64 package
+- Two types of extension resources:
+
+    - Arduino* Uno V3 connectivity
```

Comment:
Sorry, ReST is picky about indentation.  I just noticed too that these nested lists should have the indentation so the ``-`` lines up with the first letter of the parent list item or preceding sentence (otherwise it adds an extra blank line in the rendered output):

    - Two types of extension resources:

      - Arduino ...